official/docs/csharp/ApiKeys/RetrieveApiKey.cs

```diff
@@ -0,0 +1,7 @@
+using EasyPost;
+
+EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");
+
+ApiKeys api_keys = await ApiKeys.All();
+
+Console.Write(api_keys);
```

Comment:
Let's replace all the write lines with this (make sure the variables stay the same!): 

```suggestion
Console.WriteLine(JsonConvert.SerializeObject(api_keys, Formatting.Indented));
```